dbms/src/Storages/FormatVersion.h

```diff
@@ -132,10 +134,10 @@ inline static const StorageFormatVersion STORAGE_FORMAT_V5 = StorageFormatVersio
 };
 
 inline static const StorageFormatVersion STORAGE_FORMAT_V6 = StorageFormatVersion{
-    .segment = SegmentFormat::V2,
+    .segment = SegmentFormat::V3, // diff
```

Comment:
We should use STORAGE_FORMAT_V7/STORAGE_FORMAT_V102 for these changes. Because V6/V101 has been set as the default format in serverless env. And we need make them upgrade from V6/V101 to V7/V102.

Reply:
done